ntia_conformance_checker/cli_tools/check_anything.py

```diff
@@ -50,69 +53,175 @@ def check_minimum_elements(file, messages=None):
 
 
 def check_document_version(doc, messages):
+    """Check the SPDX document version.
+
+    Args:
+        file (str) - full filepath
+        messages (list) - set of messages related to minumum elements check
+
+    Returns:
+        messages (list) - set of messages related to minumum elements check
+    """
     if str(doc.version) != "SPDX-2.2" and str(doc.version) != "SPDX-2.3":
         messages.append("Document  version " + str(doc.version) + " not supported.")
-        return messages
+    return messages
 
 
 def check_components_names(doc, messages):
+    """Check that components have names.
+
+    Args:
+        file (str) - full filepath
+        messages (list) - set of messages related to minumum elements check
+
+    Returns:
+        messages (list) - set of messages related to minumum elements check
+    """
     for package in doc.packages:
         messages = check_name(package, messages)
+    return messages
 
 
 def check_components_versions(doc, messages):
+    """Check that components have versions.
+
+    Args:
+        file (str) - full filepath
+        messages (list) - set of messages related to minumum elements check
+
+    Returns:
+        messages (list) - set of messages related to minumum elements check
+    """
     for package in doc.packages:
-        if has_version(package) == False:
+        if has_version(package) is False:
             messages.append(str(package.name) + " has no version.")
 
 
 def check_sbom_author(doc, messages):
-    for i in range(len(doc.creation_info.creators)):
-        if type(doc.creation_info.creators[i]) == spdx.creationinfo.Person:
+    """Check that SBOM has an author.
+
+    Args:
+        file (str) - full filepath
+        messages (list) - set of messages related to minumum elements check
+
+    Returns:
+        messages (list) - set of messages related to minumum elements check
+    """
+    for i, _ in enumerate(doc.creation_info.creators):
+        if isinstance(doc.creation_info.creators[i], spdx.creationinfo.Person):
```

Comment:
This isn't an issue introduced with this fix, but I think an organization should be allowed for the SBOM.  I'll create a separate issue.

Reply:
Nice. Sounds good!